bindings/python/iota_sdk/types/address.py

```diff
@@ -59,6 +63,40 @@ class NFTAddress:
     type: int = field(default_factory=lambda: int(AddressType.NFT), init=False)
 
 
+@json
+@dataclass
+class ImplicitAccountCreationAddress:
+    """Represents an implicit account creation address that can be used to transition an account.
```

Comment:
Where is this doc from? It's not ideal

Reply:
I probably just wrote it, I don't remember

Reply:
Can you take from the TIP please? This is a bit unclear/incorrect

Reply:
Let me have a look quickly

Reply:
TIP says `Defines an address on which an Implicit Account is created when it receives a Basic Output`

Reply:
That's not great either :/

Reply:
I think it's slightly better, `can be used to transition an account` is not completely correct IMO

Reply:
Can you do this please?